traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -108,6 +108,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodPost, `cachegroups/{id}/queue_update$`, cachegroup.QueueUpdates(d.DB.DB), auth.PrivLevelOperations, Authenticated, nil},
 
 		//CDN
+		{1.1, http.MethodGet, `cdns/name/{name}/sslkeys/?(\.json)?$`, cdn.GetSSLKeys, auth.PrivLevelOperations, Authenticated, nil},
```

Comment:
i think this is supposed to be admin priv level

Reply:
@mitchell852 Fixed